memgpt/server/server.py

```diff
@@ -889,7 +889,7 @@ def get_blocks(
         self,
         user_id: Optional[str] = None,
         label: Optional[str] = None,
-        template: Optional[bool] = True,
+        template: Optional[bool] = None,
```

Comment:
Is this actually support be be optional bool?

As in we expect `None`, `False`, or `True` (and `None` by default)?

Or is this meant to be `bool = True`, as in `False` or `True` (no `None`), but `False` by default?

Reply:
Yeah, I was having issues with getting confused by the defaults being passed - and we also do filtering based on what values are non-null.